src/helpers/utils.ts

```diff
@@ -233,3 +233,163 @@ export const getUnClaimReward = (
         : [],
   };
 };
+
+// update reward_per_token_store
+// refer this implementation in program: https://github.com/MeteoraAg/damm-v2/blob/main/programs/cp-amm/src/state/pool.rs#L256-L281
```

Comment:
We should refer to the commit hash instead of the main branch, as it can change over time